http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -45,18 +45,15 @@ final case class ClusterRecord(id: Long,
                                customClusterEnvironmentVariables: Map[String, String],
                                runtimeConfigId: RuntimeConfigId)
 
-final case class ServiceAccountInfoRecord(clusterServiceAccount: Option[String],
-                                          notebookServiceAccount: Option[String],
-                                          serviceAccountKeyId: Option[String])
+final case class ServiceAccountInfoRecord(clusterServiceAccount: WorkbenchEmail, serviceAccountKeyId: Option[String])
```

Comment:
Hm.. can we get rid of `serviceAccountKeyId` as well? I don't think we need to care about keys if not using `notebookServiceAccount`